### ISSUE TITLE
biterateselect didn't work, and  I used red5 as my video server.

### ISSUE THREAD

**Issue by abc1225** [flowplayer/flash#241]
hello,  when I clicked the HD button, the red5 server report error:  Method play2 with parameters[..] not found in org.red5.demos.oflaDemo.Application@2fc5809e.
Did it mean i need to write some functions in my red5 server application? 

Thanks!

**Comment by danrossi** [comment]
I don't think they have dynamic stream switch support yet. I will ask them of the status or test it out myself.

**Comment by abc1225** [comment]
Thanks for your help.
The flash flowplayer can switch the stream dynamically,   the demo is at: http://flash.flowplayer.org/demos/plugins/streaming/hdtoggle.html   My video server is red5 and it can support the streaming video,  I don't know if I need do something at my server end when use the biterateselect plugin.

**Comment by danrossi** [comment]
i'll update my red5 instance but I don't believe it handles stream switching again i'll have to test it.

**Comment by abc1225** [comment]
thanks, i'm glad to waiting for your results.

**Comment by danrossi** [comment]
https://github.com/Red5/red5-server get the latest sources could be a start.

**Comment by danrossi** [comment]
it doesn't seem to be working at all. it will completely stall or trigger stop. I suppose that's what you are seeing.

**Comment by danrossi** [comment]
with the latest server code. oflaDemo is old. The new demo app is called vod.

**Comment by abc1225** [comment]
yes,  i will try the vod app with the latest server code. Should i do some operation on red5 server when i use the biterateselect plugin? Thanks.

**Comment by danrossi** [comment]
it simply doesn't work i'm afraid I just tested it.

**Comment by abc1225** [comment]
but the demo http://flash.flowplayer.org/demos/plugins/streaming/hdtoggle.html is really work well tested. how can it work?

**Comment by danrossi** [comment]
with wowza , flash media server, HDS etc.

**Comment by abc1225** [comment]
oh, got it

**Comment by danrossi** [comment]
I did a bit of debugging myself and found one particular issue but it would take time to debug that further. after switching it just hangs and it shouldn't be doing it instantly so a bit of work is still required there. I alerted the team about it.

**Comment by abc1225** [comment]
thanks. i will follow your team.

**Comment by abc1225** [comment]
hi, danrossi. 
I have deployed the development environment and configured the files(build.properties and plugin- build.properties in flash-build) according to this page   http://flash.flowplayer.org/documentation/developer/development-environment.html , but when I compile the build.xml, there was an error reported: D:\environment\flow0\flash-build\build.xml:12: D:\environment\flow0\tld\secondary doesn't exist.    Is the guidance(in the above page) not suitable to version 3.2.18?  Thanks!
Here is my directory:  
flash

```
core
js-plugins
lib
plugins
```

flash-build

```
3.2.15.properties
3.2.16.properties
3.2.17-dev.properties
3.2.17.properties
3.2.18.properties
build-anttask.xml
build.properties
build.xml
js.xml
lib
manifest-old.xml
manifest.xml
plugin-build.properties
plugin-build.xml
release.xml
src
```

site

```
content
deploy
```

**Comment by abc1225** [comment]
the flash 、 flash-build and site are the parent folder

**Comment by danrossi** [comment]
you need to grab the tld checkout from the github sources also.

**Comment by abc1225** [comment]
thanks, i compiled it successed.

**Comment by danrossi** [comment]
may I close this for now ? I will have to update once red5 have switching working

**Comment by abc1225** [comment]
yes, of course, thanks for your work.
